### PR TITLE
Datahub

### PR DESCRIPTION
**Pull request by paulstgermain**
## Description

This pull request implements the routing for the new homepage (The 'DataHub' component). It also adds the basic structuring of the component itself and a few place-holder data visualizations. These visualizations are currently built off of the same data from our database - this is something I aim to change within the next day or so after I get clarification on what data exactly would be most useful to represent there.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/DataHub/DataHub.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+      <div className="subChartsContainer">
+        <CaseDataChart />
+        <CaseDataChart />
+      </div>
+    </div>
+  );
+};
+
+export default DataHub;
```
Comment (katieolson84) [inline src/components/pages/DataHub/DataHub.js:104]: New code looks so clean and organized, great job!

File: src/components/pages/DataHub/DataHub.less [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.mainChartContainer {
+    display: flex;
+    justify-content: center;
+}
+
+.subChartsContainer {
+    display: flex;
+    justify-content: space-evenly;
```
Comment (katieolson84) [inline src/components/pages/DataHub/DataHub.less:8]: Nice job continuing to use less!

**Review by katieolson84 — APPROVED** [review]
Everything looks great. The /cases route should fit nicely with the button on the sidebar nav. I approved since I do not see any problems with this code.

**Review by MichelleCCodes — APPROVED** [review]
"DataHub"- what a cool name! Prop passing, naming, and routing looks amazing. Nice!!

**Pull request merged**